aircan/dags/api_ckan_import_to_bq.py

```diff
@@ -48,11 +48,14 @@
 
 def task_import_resource_to_bq(**context):
     logging.info('Invoking import resource to bigquery')
-    gc_file_url = context['params'].get('resource', {}).get('path')
+    logging.info("resource: {}".format(context['params'].get('resource', {})))
+
+    gc_file_url = context['params'].get('big_query', {}).get('gcs_uri')
     bq_project_id = context['params'].get('big_query', {}).get('bq_project_id')
     bq_dataset_id = context['params'].get('big_query', {}).get('bq_dataset_id')
-    bq_table_name = gc_file_url.split("/")[-1].partition('.')[0].replace('-', '_')
 
+    bq_table_name = context['params'].get('resource', {}).get('ckan_resource_id').replace('-', '')
```

Comment:
I think we need to think about this a bit more systematically and this will, i suspect, break the API instructions in the  UI (which run off resource name or similar). In general I don't like resource_id as it is uuid but maybe it is the best thing we have for now.

Reply:
Agree, this is not ideal, but `The table name must be unique per dataset`, so except the `resource_id` we don't have other property to ensure uniqueness of BQ table name. In order not to break API usage, we must have `bq_table_name` property in resource with `resource_id` (without "-" ) by one of the options Anuar suggested here - https://github.com/datopian/datapub/issues/31#issuecomment-694126659